src/openforms/registrations/contrib/json_dump/config.py

```diff
@@ -47,6 +47,32 @@ class JSONDumpOptionsSerializer(JsonSchemaSerializerMixin, serializers.Serialize
         required=True,
         min_length=1,
     )
+    fixed_metadata_variables = serializers.ListField(
```

Comment:
self-reminder to ask Joeri's thoughts about this

Reply:
its fine like this. no data migration when this changes.